content/projects/gsoc/2019/project-ideas/artifactory-rest-plugin.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: gsocprojectidea
+title: "Artifactory REST Plugin"
+goal: "Create a new plugin to give Jenkins users the ability to make REST API calls to Artifactory natively"
+category: Plugin
+year: 2019
+status: draft
+showGoogleDoc: true
+sig: gsoc
+skills:
+- Java
+- REST API
+- Artifactory
+- Jenkins Pipeline
+mentors:
+- name: "Martin d'Anjou"
+  id: "deepchip"
+  github: "martinda"
+- name: "Christopher Dancy"
+  github: "cdancy"
+links:
+  gitter: "jenkinsci/gsoc-sig"
```

Comment:
No need to reference it now, it will be picked from the SIG metadata automatically